### ISSUE TITLE
Library outdated

### ISSUE THREAD

**Issue by eytanbiala** [helpscout/helpscout-api-java#1]
I am finding many inconsistencies between this library and the current version of the API.

For example, the Webhook class is looking for headers that do not exist, such as "HTTP_X_HELPSCOUT_EVENT", which now seems to be "X-Helpscout-Signature". 
So, the header returns null and every webhook request is considered invalid when using the isValid() method.

Also, the Parser class does not work correctly. For example, the Customer object is always null in this example for a Dynamic app:

``` java
Webhook webhook = new Webhook(SECRET_KEY, request);
Customer customer = webhook.getCustomer(); 
```

**Comment by jacobwallace** [comment]
@eytanbiala I don't have an ETA at the moment, but it is on our roadmap to fix this issue. Thanks for the feedback!

**Comment by eytanbiala** [comment]
No problem - thanks for acknowledging the issue in any case.

**Comment by bob983** [comment]
Hi @eytanbiala  It's been a while, but @sphinks fixed the header issue in #20. I'm going to update documentation, changelog and release new version.
